packages/unmock-core/src/service/parser.ts

```diff
@@ -0,0 +1,34 @@
+import jsYaml from "js-yaml";
+import { IServiceDef, IServiceDefFile, IServiceParser } from "../interfaces";
+import { Service } from "./service";
+
+const PATTERN_FOR_KNOWN_FILENAMES = /^(?:index|spec|openapi)\.ya?ml$/;
+
+export class ServiceParser implements IServiceParser {
+  public parse(serviceDef: IServiceDef): Service {
+    const name = serviceDef.directoryName;
+    const serviceFiles = serviceDef.serviceFiles;
+
+    const matchingFiles = serviceFiles.filter(
+      (serviceDefFile: IServiceDefFile) =>
+        PATTERN_FOR_KNOWN_FILENAMES.test(serviceDefFile.basename),
+    );
+
+    if (matchingFiles.length === 0) {
+      throw new Error(
+        `No idea what to do with: ${JSON.stringify(serviceFiles)}`,
```

Comment:
Maybe a more indicative error message here? `Can't locate service specification file with pattern ... for ...` or something?